priv/request.ex.eex

```diff
@@ -15,7 +15,7 @@ defmodule GenLSP.Requests.<%= LSPCodegen.Naming.name(@request) %> do<%= if @requ
     field :method, String.t(), default: "<%= @request.method %>"
     field :jsonrpc, String.t(), default: "2.0"
     field :id, integer(), enforce: true
-    field :params, <%= LSPCodegen.Codegen.to_string(@params, @metamodel) %>
+    <%= unless @params == :none do %> field :params, <%= LSPCodegen.Codegen.to_string(@params, @metamodel) %> <% end %>
```

Comment:
not sure if there is a way to have the condition on separate lines without forcing a space.